openslides/assignments/static/js/assignments/site.js

```diff
@@ -38,11 +38,11 @@ angular.module('OpenSlidesApp.assignments.site', [
             .state('assignments.assignment.list', {
                 resolve: {
                     assignments: function(Assignment) {
-                        return Assignment.findAll();
-                    },
-                    users: function(User) {
-                        //TODO: Load only related users cause of better performance
-                        return User.findAll();
+                        return Assignment.findAll().then(function(assignments) {
+                            angular.forEach(assignments, function(assignment) {
+                                Assignment.loadRelations(assignment, ['assignment_related_users']);
```

Comment:
@normanjaeckel Same error if I use this line to load all related users. Any idea?